community/user-stories/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: single
+title:  "Spinnaker User Stories"
+sidebar:
+  nav: community
+redirect_from: /community/user-stories
+---
+
```

Comment:
I would add `{% include toc %}` here, just so we see at the top that there are two sections--users/evaluators and builders/supporters.

Reply:
@dorbin I originally had it on, but it covers up the first two videos: here's a screenshot: http://drod.io/8f610850ba89 and the youtube sizing on embeds doesn't work so I figured the simpler thing was to just remove the TOC. Open to suggestions, though.